src/demo/index.html

```diff
@@ -105,6 +105,16 @@ <h1>Symphony Electron API Demo</h1>
         <p>snippet output:</p>
         <image id='snippet-img'/>
 
+        <hr>
+        <p>RTC Logs:</p>
```

Comment:
I think you should remove RTC from the example. Both from the text and from the element IDs. This feature is unaware who is adding logs. Name it something generic instead.

Reply:
Will do